src/action-creators/importText.js

```diff
@@ -0,0 +1,99 @@
+// util
+import {
+  contextOf,
+  head,
+  importHtml,
+  pathToContext,
+  rawTextToHtml,
+  rootedContextOf,
+  strip,
+} from '../util'
+
+// a list item tag
+const regexpListItem = /<li(?:\s|>)/gmi
+
+/** Imports the given text or html into the given thoughts
+ *
+    @param preventSetCursor  Prevents the default behavior of setting the cursor to the last thought at the first level
+    @param preventSync       Prevent syncing state, turning this into a pure function.
+    @param rawDestValue      When pasting after whitespace, e.g. pasting "b" after "a ", the normal destValue has already been trimmed, which would result in "ab". We need to pass the untrimmed destination value in so that it can be trimmed after concatenation.
+    @param skipRoot          See importHtml @param.
+ */
+const importText = (thoughtsRanked, inputText, { preventSetCursor, preventSync, rawDestValue, skipRoot } = {}) => (dispatch, getState) => {
+  const text = rawTextToHtml(inputText)
```

Comment:
Since `rawTextToHtml` is only used in one place, you should define it internally to the module. That helps to reduce complexity and minimize the surface area of the inter-module API.

Reply:
Sure!